src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -251,6 +250,30 @@ export class CalciteCombobox {
   /** Called when a selected item in the combobox is dismissed via its chip **/
   @Event() calciteComboboxChipDismiss: EventEmitter;
 
+  /**
+   * Fired when the combobox is opened
+   * @internal
+   */
+  @Event() calciteComboboxOpen: EventEmitter;
+
+  /**
+   * Fired when the combobox is opened and the transition has ended
+   * @internal
+   */
+  @Event() calciteComboboxOpenEnd: EventEmitter;
+
+  /**
+   *  Fired when the combobox is closed
+   * @internal
+   */
+  @Event() calciteComboboxClose: EventEmitter;
+
+  /**
+   *  Fired when the combobox is closed and the transition has ended
+   * @internal
+   */
+  @Event() calciteComboboxCloseEnd: EventEmitter;
```

Comment:
Since these prop events have animation tied to them, can they be merged and have them fire after the animation/transition is complete (e.g., only `close` vs `close` + `closeEnd`)? In this particular case, the combobox is truly closed after the animation is done.

Reply:
@jcfranco in this case, since both are internal we could do this. However, in some of the other cases there are already events that emit immediately and we can't change them to fire after animation without it being a breaking change that requires updates to tests. 

For example, the following is expected to happen immediately in the tests.

```
/** Fired when the popover is opened */
  @Event() calcitePopoverOpen: EventEmitter;
```

Do we want to change that?

Reply:
Same with `calciteInlineEditableEditingCancel`

Reply:
> breaking change that requires updates to tests.

I don't see this is as a breaking change. Currently, when this event fires the popover isn't truly open until the transition is done, right?

Reply:
The tests would need to be updated to wait for the transition then.